src/Eppie.CLI/Eppie.CLI/Menu/Actions.cs

```diff
@@ -299,29 +305,94 @@ private async Task AddEmailAccountAsync()
         {
             _logger.LogMethodCall();
 
-            Account account = await GetDefaultAccountAsync().ConfigureAwait(false);
-            await _coreProvider.TuviMailCore.AddAccountAsync(account).ConfigureAwait(false);
+            async Task<Account> CreateAccountAsync()
+            {
+                MailService mailService = _application.SelectOption(MailService.Other, true);
+
+                return mailService == MailService.Other ? CreateDefaultAccount()
+                                                        : await CreateOAuth2AccountAsync(mailService).ConfigureAwait(false);
+            }
+
+            try
+            {
+                Account account = await CreateAccountAsync().ConfigureAwait(false);
+                await _coreProvider.TuviMailCore.AddAccountAsync(account).ConfigureAwait(false);
+            }
+            catch (OperationCanceledException)
+            {
+                //ToDo: Move string to resources
```

Comment:
Could you please file an issue for all tasks of moving strings to resources?

Reply:
Issue #108 is open.